src/spikeinterface/core/recording_tools.py

```diff
@@ -14,6 +14,7 @@ def get_random_data_chunks(
     concatenated=True,
     seed=0,
     margin_frames=0,
+    job_kwargs={},
```

Comment:
I would prefer having `**job_kwargs` and automatically populate the dosctring and propagate the arguments.. no?

Reply:
Also, remember that setting default values with mutable objects is a bad practice that lends itself to hard to debug errors:
https://nedbatchelder.com/blog/202311/say_it_again_values_not_expressions.html

Reply:
Thanks for the feedback, that I'll consider asap. For now this is just a draft, because I realized that several options of the current API (no concatenation / margin frames) will not be straightforward with this current implementation. However, if there is an interest this is doable.